mlibrary/my_account
# February 2023 dependency updates.

# Overview
Monthly dependency updates to help maintain Account.

`sass`, `eslint`, and `eslint` related dependencies have been updates.

## `npm` Update?
While building the css (`docker-compose run --rm web npm run build-css`), this message popped up:

```
New major version of npm available! 8.19.1 -> 9.6.0
Changelog: https://github.com/npm/cli/releases/tag/v9.4.2
Run npm install -g npm@9.4.2 to update!
```

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Build the CSS (docker-compose run --rm web npm run build-css).
- Go through [localhost](http://localhost:4567/) and make sure nothing is borked.
